src/app/learn/accessibility/design/checklist/index.html

```diff
@@ -0,0 +1,182 @@
+<stache
```

Comment:
@Blackbaud-MattGregg, I converted the "SKY UX Accessibility checklist for designers v1.doc" into this index.html file. Can you please check behind me on this?